source/content/global-cdn.md

```diff
@@ -52,36 +52,67 @@ Users with session-style cookies set, or a `NO_CACHE` cookie set will bypass the
 
 ### Confirm That Persistent Cache Works
 
-To test how stale cache is served, compare the header results of a page refresh of the site's Dev environment to the same page in Maintenance Mode.
-
-Navigate to the page using [Firefox](https://developer.mozilla.org/en-US/docs/Tools) or [Chrome](https://developer.chrome.com/docs/devtools/), and in the browser's developer tools open the **Network** tab and view the response headers for the page or asset. Refresh the page in the browser for the newest information.
-
-Examine the results for the `age` and `max-age`.
-
-To examine the headers through the command line, use a `curl` command to examine the headers before and after you set the site to Maintenance Mode:
-
-```bash{outputLines: 2-20}
-curl --head https://pantheon.io/docs
-HTTP/2 301
-content-type: text/html
-location: https://pantheon.io/docs/
-server: nginx
-strict-transport-security: max-age=31622400
-x-pantheon-styx-hostname: styx-fe2-a-5d96768699-vcdvh
-x-styx-req-id: b7b8d4d2-04d9-11ec-a467-9a05fab906d1
-cache-control: public, max-age=86400
-date: Tue, 24 Aug 2021 15:30:21 GMT
-x-served-by: cache-mdw17379-MDW, cache-ewr18124-EWR
-x-cache: HIT, HIT
-x-cache-hits: 1, 1
-x-timer: S1629819022.932985,VS0,VE1
-pantheon-trace-id: be58e6a03a904fbfa64515ee136ffd34
-vary: Cookie, Cookie
-age: 9654
-accept-ranges: bytes
-via: 1.1 varnish, 1.1 varnish
-content-length: 162
-```
+To test how stale cache is served, compare the header results of a page refresh:
+
+<TabList>
+
+<Tab title="Via Command Line" id="cli" active={true}>
+
+1. Examine the headers through the command line:
+
+  ```bash{outputLines: 2-20}
+  curl --head https://pantheon.io/docs
+  HTTP/2 301
+  content-type: text/html
+  location: https://pantheon.io/docs/
+  server: nginx
+  strict-transport-security: max-age=31622400
+  x-pantheon-styx-hostname: styx-fe2-a-5d96768699-vcdvh
+  x-styx-req-id: b7b8d4d2-04d9-11ec-a467-9a05fab906d1
+  cache-control: public, max-age=86400
+  date: Tue, 24 Aug 2021 15:30:21 GMT
+  x-served-by: cache-mdw17379-MDW, cache-ewr18124-EWR
+  x-cache: HIT, HIT
+  x-cache-hits: 1, 1
+  x-timer: S1629819022.932985,VS0,VE1
+  pantheon-trace-id: be58e6a03a904fbfa64515ee136ffd34
+  vary: Cookie, Cookie
+  age: 9654
+  accept-ranges: bytes
+  via: 1.1 varnish, 1.1 varnish
+  content-length: 162
+  ```
+  
+  Note the result for `age` or `max-age`.
+  
+1. Navigate to your site's Dev environment and set the site to Maintenance Mode.
+  
+1. Clear the cache from either the Advanced Page Cache module or from the Dashboard.
+
+1. In a terminal, cURL the site headers filtered for stale cache:
+
+  ```bash{promptUser: user}
+  curl --head https://pantheon.io/docs | grep PContext-Resp-Is-StaleHTTP/2 301
+  ```  
+  If the result includes `PContext-Resp-Is-StaleHTTP/2 301`, the page has been successfully served from stale cache. 
```

Comment:
```suggestion
  If the result includes `PContext-Resp-Is-Stale`, the page has been successfully served from stale cache. 
```
The HTTP/2 part shouldn't appear like that, it may be your terminal interleaving curl's stdout and stderr?

Try
```bash{promptUser: user}
  curl -v https://pantheon.io/docs 2>&1 | grep PContext-Resp-Is-Stale
  ```